data/human/ships.txt

```diff
@@ -1260,6 +1260,63 @@ ship "Cruiser"
 
 
 
+ship "Cutthroat"
+	sprite "ship/pirate cutthroat"
+	thumbnail "thumbnail/pirate cutthroat"
+	attributes
+		category "Light Warship"
+		"cost" 1100000
+		"shields" 2800
+		"hull" 600
+		"required crew" 4
+		"bunks" 6
+		"mass" 171
```

Comment:
```suggestion
		"mass" 185
```
An extra 21 mass does not seem nearly enough to account for a 38% increase in engine space, especially since this ship is 21 lighter thanks to the cargo-to-outfit space conversion. This 21 merely accounts for the change in the outfitting space, and should have significantly more to account for the two big engine nacelles bolted onto the wings.

Reply:
Note they also did carve out bunks to upgrade so it's not just cargo space to outfit space but include the bunk area as well. The mass I added was accounting mostly for the added on engine pods.